### PR TITLE
feat: add border-radius style for elements

### LINKED ISSUE #18063: Add function to set borderRadius
### Describe your motivation

add function to set borderRadius inside Style class

It's possible if you use the ```set("","");```
but it's a bit complicated

I want this function to be easier, and it can also add completeness to the functions in the Vaadin Style class
vaadin version:24.2.3
### Describe the solution you'd like

```
getStyle().setBorderRadius("5px");
```

### PR DESCRIPTION
**Pull request by Rystars**
## Description

Fixes #18063

## Type of change

- [ ] Bugfix
- [x] Feature

## Checklist

- [x] I have read the contribution guide: https://vaadin.com/docs/latest/guide/contributing/overview/
- [x] I have added a description following the guideline.
- [x] The issue is created in the corresponding repository and I have referenced it.
- [ ] I have added tests to ensure my change is effective and works as intended.
- [ ] New and existing tests are passing locally with my change.
- [ ] I have performed self-review and corrected misspellings.

#### Additional for `Feature` type of change

- [x] Enhancement / new feature was discussed in a corresponding GitHub issue and Acceptance Criteria were created.

### CODE REVIEW

**Review by mcollovati — APPROVED** [review]

**Comment by mcollovati** [comment]
@Rystars thank you for the contribution!
Could you please format the source code as described in [this comment](https://github.com/vaadin/flow/pull/18089#issuecomment-1821175393)?

File: .github/workflows/maven_formatter.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# This workflow will build a Java project with Maven, and cache/restore any dependencies to improve the workflow execution time
```
Comment (mcollovati) [inline .github/workflows/maven_formatter.yml:1]: Please, remove this file that is not related to this PR

**Review by mcollovati — CHANGES_REQUESTED** [review]

**Comment by Rystars** [comment]
> @Rystars thank you for the contribution! Could you please format the source code as described in [this comment](https://github.com/vaadin/flow/pull/18089#issuecomment-1821175393)?

Sorry I can only answer because I was asleep yesterday

the error format has been fixed (maybe)

File: .github/workflows/maven_formatter.yml (hunk 1 above)
Comment (Rystars) [reply .github/workflows/maven_formatter.yml]: It's been deleted but why is it still there?🤔

File: flow-server/src/main/java/com/vaadin/flow/dom/Style.java [hunk 2]
```diff
@@ -183,6 +183,18 @@ default Style setBorderBottom(String value) {
...
+    /**
+     * Sets the <code>border-radius</code> property.
+     *
+     * @param value
+     *            the style property value (if <code>null</code>, the property
+     *            will be removed)
+     * @return this style instance
+     */
+    default Style setBorderRadius(String value) {
+        return set(STYLE_BORDER_RADIUS,value);
```
Comment (mcollovati) [inline flow-server/src/main/java/com/vaadin/flow/dom/Style.java:195]: Fixing formatting issue

```suggestion
        return set(STYLE_BORDER_RADIUS, value);
```

**Review by mcollovati — CHANGES_REQUESTED** [review]

**Review by mcollovati — APPROVED** [review]

**Comment by mcollovati** [comment]
@Rystars there was still a formatting issue, but the PR now looks good. Thank you for the contribution!

**Comment by Rystars** [comment]
thanks for fixing it

**Pull request merged**